build.sbt

```diff
@@ -88,11 +88,6 @@ lazy val nativelib =
   project.in(file("nativelib")).
     settings(libSettings)
 
-lazy val clib =
-  project.in(file("clib")).
-    settings(libSettings).
-    dependsOn(nativelib)
-
```

Comment:
Splitting out libc out of nativelib was a mistake. It added one extra full import to reach common functionality for no good reason.